build/build-report.js

```diff
@@ -60,11 +60,14 @@ async function buildFlowReport() {
     plugins: [
       rollupPlugins.inlineFs({verbose: true}),
       rollupPlugins.replace({
-        '__dirname': '""',
+        delimiters: ['', ''],
+        values: {
+          'getModuleDirectory(import.meta)': '""',
+        },
```

Comment:
inline-fs replaces the entire expression using `moduleDir` anyway, so this doesn't matter ... except to remove invalid syntax/code from the bundled result. Ideally, a bundler would drop `const moduleDir = ...` entirely (and the import of `getModuleDirectory` entirely, since unused) ... but rollup isn't doing that for us. Maybe esbuild would :) 

Or maybe it just need some sort of `pure` comment? idk. for now, easier to just rip it out like this.